### ISSUE TITLE
`Bug` During offline mode I can not change the temperature of the thermostat

### ISSUE THREAD

**Issue by itSme1990** [Blackymas/NSPanel_HA_Blueprint#2369]
### TFT Version

4.3.11

### Firmware Version

4.3.11

### Blueprint Version

4.3.11

### Panel Model

EU

### What is the bug?

I use the climate add-on with relay 1 to control an underfloor heating. If home assistant is online everything works fine. But during offline mode I can not change the temperature of the thermostat. If I click on the current temperature in home screens it does not show the thermostat page.
Btw.: It still controls the temperature but I can just not change the target temperature.
I am not sure if this is a bug or works as expected. If it is not a bug than I would rephrase it as a feature request :)

### Steps to Reproduce

* Activate the climate add-on in the esp configuration
* Shut down Home assistant.
* Click on the temperature on Home screen

**Comment by edwardtfn** [comment]
> I am not sure if this is a bug or works as expected.

I was expecting the local control of target temperature even with Wi-Fi out, so certainly a bug.

**Comment by itSme1990** [comment]
Thanks for your fast response and great work! If you need any logs or something else, please let me know :)

**Comment by edwardtfn** [comment]
I will try to duplicate it here. If I cannot duplicate, then I will ask for additional info.

**Comment by Nevinskas** [comment]
I just want to double confirm, that I'm also having the same issue.
Let me know as well if you need more debug info for this problem.
Thanks!

**Comment by Nevinskas** [comment]
Hey @itSme1990 @edwardtfn 
I've made myself a setup for debugging this problem. Have reproduced it and have some info.
@itSme1990 Is it okay if I post my findings here? I don't want to hijack someones else problem. 
Or should I create separate issue and link with this one?
Thanks!

**Comment by itSme1990** [comment]
@Nevinskas, yes sure - Please post it here. Great to hear you do have a debugging setup for the issue 🙂

**Comment by edwardtfn** [comment]
That will be very useful. 😃

**Comment by Nevinskas** [comment]
hey @edwardtfn 
so finally I found some spare time to log everything.
I've compiled fw with verbose logging, and attaching full logs for each finding.
This might be overwhelming at first, but I think I found few problem related:

1. While the NSPanel is offline, pressing on the temperature numbers nothing happens. 
[pressing_on_temperature.log](https://github.com/user-attachments/files/18726395/pressing_on_temperature.log) 
I think this is initial issue. I've marked the places in the log with XXXX where I did pressing.
But after digging deeper, I've found that you can get into thermostat dialog by long pressing the temperature numbers. @itSme1990 Check if you can get in to the thermostat setting while the NSPanel is offline by pressing and holding on temperature. 

2. While the NSPanel is offline, changing temperature (sometimes) causes to crash the NSPanel.  [crash_on_changing_temperature.log](https://github.com/user-attachments/files/18726396/crash_on_changing_temperature.log) 
I've got this crash by getting in to the thermostat settings and changing the temperature. I haven't reproduced it the second time, but I thought that someone else might find it interesting.

3. While the NSPanel is offline, turning up the heat to trigger the heating relay, the NSPanel crashes. 
[random_crash_after_turning_heating_on_2.log](https://github.com/user-attachments/files/18726399/random_crash_after_turning_heating_on_2.log)
[random_crash_after_turning_heating_on.log](https://github.com/user-attachments/files/18726398/random_crash_after_turning_heating_on.log)
So, when the NSPanel triggers the relay for a heating, after a period of time (5-20secs) the panel crashes. Got this reproduced two times and both logs attached.

4. After some crash, the NSPanel does not responds to any "commands".
[reboot_not_working_and_holding_on_temp.log](https://github.com/user-attachments/files/18726460/reboot_not_working_and_holding_on_temp.log)
The NSPanel got into weird state, where the screen is showing (and touch is working), but getting into thermostat dialog page is impossible by holding finger on temperature numbers. Also, by holding down the time, you can get into different dialogue (with brightness and reboot button), but reboot button does not work. You have to power cycle the panel.
This is most likely not related to the thermostat, but I've think you might find this interesting as well.

I would attach the firmware elf file (for gdb debugging) but it contains my wifi password. But if you think that the fw file would help to debug this, I would not mind sharing the fw elf via email.
Also, let me know if you need something to clear up or attach additional info.

**Comment by FelixTechgiti** [comment]
Hi, I am facing the same situation.

I configured the panel with the climate_heat add-on to ensure local control.
However, when Home Assistant is offline (or stopped), the Climate Page on the panel loses all text info (current temp, target temp, state) and only shows the slider graphic.

Since the logic is running locally on the ESP32, I would expect the display to show the local values even without a server connection. Currently, I have no visual feedback on the device state or the set temperature when HA is unavailable.

Is there a way to configure the blueprint so that the Climate Page retrieves its values directly from the local ESPHome entity instead of waiting for HA?
